CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ## Unreleased
 
 * Compile JavaScript with Babel for browsers that don't support ES6
+* If the root dir contains directories that look like `bower_components-${foo}`, they will be treated as `variant dependencies`. In that case, polyserve will start one server for each variant, enabling testing and development of your code against each set of dependencies.
```

Comment:
WDYT of the phrase "dependency variants" and here, "treated as a `dependency variant`" ?